src/pages/ReviewResults.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {div, h} from 'react-hyperscript-helpers';
+import {AccessReviewHeader} from './access_review/AccessReviewHeader';
+import {DAR, DataSet, Election, Researcher} from '../libs/ajax';
+import {Notifications} from '../libs/utils';
+import * as fp from 'lodash/fp';
+import {DarApplication} from "./access_review/DarApplication";
+
+const SECTION = {
+  margin: '20px',
+};
+
+class ReviewResults extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  componentDidMount()  {
+    const darId = this.props.match.params.referenceId;
+    this.getData(darId);
+  }
+
+  async getData(darId) {
+
+    //get information on datasets, consent, researcher, and access request
+    const getDarData = async (darId) => {
+      let datasets;
+      let darInfo;
+      let consent;
+      let researcherProfile;
+
+      try {
+        darInfo = await DAR.getPartialDarRequest(darId);
+        const researcherPromise = await Researcher.getResearcherProfile(darInfo.userId);
+        const datasetsPromise = darInfo.datasetIds.map((id) => {
+          return DataSet.getDataSetsByDatasetId(id);
+        });
+        const consentPromise = await DAR.getDarConsent(darId);
+        [consent, datasets, researcherProfile] = await Promise.all([
+          consentPromise,
+          Promise.all(datasetsPromise),
+          researcherPromise
+        ]);
+
+      } catch (error) {
+        Notifications.showError({text: 'Error initializing DAR Data'});
+        return Promise.reject(error);
+      }
+
+      return {datasets, darInfo, consent, researcherProfile};
+    };
+
+
+    //get information on access election and research purpose election
+    const getElectionInformation = async (darId) => {
+      let accessElection;
+      let accessElectionReview;
+      let rpElectionReview;
+
+      try {
+        accessElection = await Election.findElectionByDarId(darId);
+      } catch (error) {
+        //access election is null
+        Notifications.showInformation({text: 'There is not an active data access election for this DAR.'});
```

Comment:
This alert shows up in the Reviewed Cases link which doesn't make sense - we already expect that there's no active election for the DAR.